src/frontend/src/pages/ProjectDetailPage/ProjectForm/ProjectFormDetails.tsx

```diff
@@ -55,22 +59,26 @@ const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({
             <ChangeRequestDropdown control={control} name="crId" />
```

Comment:
can you make these fields take up more room if we're in edit mode so they fill out the whole row:
<img width="925" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/411b8693-4dbb-4190-b996-daf3bc1258c7">